dev-tools.sh

```diff
@@ -211,8 +237,8 @@ elif [ "$1" == "pg_upgrade" ]; then
     cd $TARGET_WS
     if [ ! -d "./upgrade" ]; then
         mkdir upgrade
-    else
-        rm upgrade/*
+    #else
+    #    rm upgrade/*
```

Comment:
Let's remove it if is not used. Btw, is it okay not to remove the existing folder?

Reply:
I have tested without removing the existing folder and worked.